pennylane/templates/embeddings/amplitude.py

```diff
@@ -27,7 +27,8 @@
 )
 from pennylane.variable import VariableRef
 
-TOLERANCE = 1e-3
+# tolerance for normalization
+TOLERANCE = 1e-10
```

Comment:
This change might be outdated if you merge the `master` branch into `add_base_templates` branch. Since there changes were committed to master, but they were not to `add_base_templates` as per this diff.